apps/apollo-stories/src/components/ContentItemDuoAction/ContentItemDuoAction.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { Meta, StoryObj } from "@storybook/react";
+import bank from "@material-symbols/svg-700/rounded/account_balance_wallet-fill.svg";
+import { ContentItemDuoAction } from "@axa-fr/design-system-apollo-react/lf";
+import { ComponentProps } from "react";
+
+const meta: Meta<typeof ContentItemDuoAction> = {
+  component: ContentItemDuoAction,
+  title: "Components/ContentItemDuoAction",
+};
+
+export default meta;
+type ContentItemDuoActionProps = ComponentProps<typeof ContentItemDuoAction>;
+
+type Story = StoryObj<ContentItemDuoActionProps>;
+
+export const Playground: Story = {
+  name: "toggle",
+  render: (props: ContentItemDuoActionProps) => (
+    <ContentItemDuoAction {...props} />
+  ),
+  args: {
+    mode: "toggle",
+    contentItemProps: {
+      type: "icon",
+      icon: bank,
+      title: "Texte principale",
+      subtitle1: "Texte secondaire",
+      subtitle2: "Texte tertiare",
+    },
+  },
+};
+
+export const PictureLStory: Story = {
+  name: "edit",
+  render: (props: ContentItemDuoActionProps) => (
+    <ContentItemDuoAction {...props} />
+  ),
+  args: {
+    mode: "edit",
+    onEditButtonClick: () => console.log("Edit button clicked"),
+    onDeleteButtonClick: () => console.log("Delete button clicked"),
+    contentItemProps: {
+      type: "icon",
+      icon: bank,
+      title: "Texte principale",
+      subtitle1: "Texte secondaire",
+      subtitle2: "Texte tertiare",
+    },
+  },
```

Comment:
Les consoles log sur les fonctions ne sont pas nécessaire.
Grâce à storybook ça peut être intégré dans l'onglet Actions en faisant comme ceci : 


```suggestion
  args: {
    mode: "edit",
    contentItemProps: {
      type: "icon",
      icon: bank,
      title: "Texte principale",
      subtitle1: "Texte secondaire",
      subtitle2: "Texte tertiare",
    },
  },
  argTypes: {
    onEditButtonClick: { action: "onEditButtonClick" },
    onDeleteButtonClick: { action: "onDeleteButtonClick" },
  }
```

Reply:
ah cool merci je ne connaissais pas !